packages/components/Menu/src/MenuItemCheckbox/MenuItemCheckbox.styling.ts

```diff
@@ -25,14 +28,37 @@ export const stylingSettings: UseStylingOptions<MenuItemCheckboxProps, MenuItemC
       }),
       ['backgroundColor', ...borderStyles.keys, ...layoutStyles.keys],
     ),
+    ...(Platform.OS === 'android' && {
+      checkbox: buildProps(
+        (tokens: MenuItemCheckboxTokens) => ({
+          style: {
+            height: tokens.checkboxSize,
+            marginEnd: tokens.paddingHorizontal,
+            width: tokens.checkboxSize,
+            backgroundColor: tokens.checkboxBackgroundColor,
+            borderColor: tokens.checkboxBorderColor,
+            borderRadius: tokens.checkboxBorderRadius,
+            borderWidth: tokens.checkboxBorderWidth,
+            alignItems: 'center',
+            justifyContent: 'center',
+          },
+          ...(hasPressRententionForA11y && {
+            pressRetentionOffset: typeof tokens.padding === 'number' ? tokens.padding : parseFloat(tokens.padding), /// Retention of the press area outside of the checkbox equal to padding to match accessibility requirement
+          }),
+          android_ripple: { color: tokens.rippleColor, radius: tokens.checkmarkSize, foreground: true },
+        }),
+        ['checkboxBackgroundColor', 'checkboxBorderColor', 'checkboxBorderRadius', 'checkboxBorderWidth', 'checkboxSize', 'rippleColor'],
+      ),
+    }),
+
     checkmark: buildProps(
       (tokens: MenuItemCheckboxTokens) => ({
         opacity: tokens.checkmarkVisibility,
-        color: tokens.color,
+        color: tokens.checkmarkColor,
```

Comment:
This is a breaking change. Can we do tokens.checkmarkColor ?? tokens.color so that it falls back to the previous behavior?